sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -324,7 +324,7 @@ queryPrimary
     ;
 
 sortItem
-    : expression ordering=(ASC | DESC)?
+    : expression ordering=(ASC | DESC)? (NULLS nullOrder=(LAST | FIRST))?
```

Comment:
The SQL standard supports ```NULLS FIRST|LAST```. And the major relational databases implement this way. Do you think we should allow `NULL FIRST|LAST` without the `s`?